tutor/src/screens/task/page-content-ux.js

```diff
@@ -7,6 +7,9 @@ export default class PageContentUX extends BookUX {
   constructor({ main }) {
     super();
     this.mainUX = main;
+    this.update({
+      courseId: this.mainUX.course.id,
```

Comment:
Don't you need to use the courseId or check that the course is set?

Reply:
no, "mainUX" is the ux on the tasks, which will always have a course